src/Error/Renderer/ConsoleExceptionRenderer.php

```diff
@@ -80,7 +80,7 @@ public function render(): ResponseInterface|string
             $out = array_merge($out, $this->renderException($error, $parent));
         }
 
-        return join("\n", $out);
+        return implode("\n", $out);
```

Comment:
Shouldn't there be a Code sniffer for this too?
I sure use one rule here for it.

Reply:
Ah, thats a custom one probably

     [x] Function name join() found, should be implode(). (PhpCollective.PHP.RemoveFunctionAlias.LongInvalid)

https://github.com/php-collective/code-sniffer/blob/master/PhpCollective/Sniffs/PHP/RemoveFunctionAliasSniff.php